recipes/nitro/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+set -x
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX ..
```

Comment:
You can use the `$SRC_DIR` to be consistent with `bld.bat`.
Also, same question about the tests.